index.html

```diff
@@ -13,13 +13,14 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
     <!-- POPULATES PROJECTDATA SECTION -->
     <script id="projectData-template" type="text/x-handlebars-template">
-      <article data-category="{{category}}">
-        <!-- <address><a href="{{repoUrl}}">{{repoUrl}}</a></address> -->
-        <h1>{{{title}}}</h1>
+      <article class="clearfix" data-author="{{author}}" data-category="{{category}}">
```

Comment:
Update handlebar template to be reflect the attributes of your portfolio items. Example: will not need author.